libs/design-system/src/cards/CardTile.tsx

```diff
@@ -1,7 +1,11 @@
 import styled from '@emotion/styled';
+import { useMantineTheme } from '@mantine/core';
 import { colors, shadows } from '../config';
 
-export const CardTile = styled.button`
+export const CardTile = styled.button(() => {
+  const theme = useMantineTheme();
+
+  return `
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This fixes the ugly dark mode issue due to a bug I've brought up with Mantine theming‏
![Screenshot 2024-04-27 at 1 13 25 PM](https://github.com/novuhq/novu/assets/47441786/7275bf9e-1455-49b9-8335-c0be0f86e2b6)